lib/tools.js

```diff
@@ -351,6 +358,20 @@ function stringify (obj) {
   }
 }
 
+function Formatters (level, bindings, log) {
+  this.level = level
+  this.bindings = bindings
+  this.log = log
+}
+
+class DeprecationWarning extends Error {
```

Comment:
I'd prefer to avoid `class`. What is wrong with a regular error?

Reply:
agreed this is unnecessary noise - because these passed to `process.emitWarning`, which just logs the error message. You don't even need an error for `process.emitWarning` using strings is fine